.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
```

Comment:
also need .env for security of pushing confidential information like token and user metadata.